### PR TITLE
Document services and docker-compose.

### PR DESCRIPTION
**Pull request by nzcodarnoc**
Remove unused links and proxy endpoints discovered during documentation.

### CODE REVIEW

**Review by JacOng17 — APPROVED** [review]
LGTM, build works & the **Regenerate docker-compose.png** works too :+1: 
image
image

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# reverse-proxy-docker-compose-express
+# Korerorero / Reverse Proxy
 
 `http-proxy-middlware` to serve a reverse proxy that routes to multiple `express` applications as containers via `docker-compose`
```
Comment (mermop) [inline README.md]:
```suggestion
`http-proxy-middleware` to serve a reverse proxy that routes to multiple `express` applications as containers via `docker-compose`
```

File: docs/request-sequence.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+:root { --mermaid-font-family: "trebuchet ms", verdana, arial;}</style><style>#mermaid-1587532740696 {
```
Comment (mermop) [inline docs/request-sequence.svg:3]: For some reason this image won't preview and the `</style>` on this line is highlighted red

File: docs/services.mmd [hunk 3]
```diff
@@ -0,0 +1,31 @@
+classDiagram
+    class reverse_proxy {
+        <<Service>> 
+        http://local-host:8000
```
Comment (mermop) [inline docs/services.mmd]: not `localhost`?

File: docs/services.mmd [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    class text_to_speech {
+        <<Service>>
+        http://tts:59125
+    }
+    reverse_proxy -- front_end
+    reverse_proxy -- orchestration
+    orchestration -- mouth_shapes
+    orchestration .. text_to_speech: API knowledge
+    text_to_speech -- mouth_shapes: tts audio is saved in mouth_shapes
+    mouth_shapes -- reverse_proxy: tts audio proxy
```
Comment (mermop) [inline docs/services.mmd:31]: This is actually a really lovely way of making these diagrams - hadn't come across it before, looks great

**Review by mermop — APPROVED** [review]
Yay!

File: docs/services.mmd (hunk 3 above)
Comment (nzcodarnoc) [reply docs/services.mmd]: Defo a typo, I've pushed a correction.

File: docs/request-sequence.svg (hunk 2 above)
Comment (nzcodarnoc) [reply docs/request-sequence.svg:3]: You can see the file renders correctly here (by navigating to the file and selecting to view 'raw'): https://raw.githubusercontent.com/ServiceInnovationLab/korerorero-reverse-proxy/87b711bfbf41440ceee500886b8c94273c3c084b/docs/request-sequence.svg

I'm going to put this down to a githubism with a complex SVG.

**Pull request merged**